webapi/src/main/scala/org/knora/webapi/messages/util/DateUtil.scala

```diff
@@ -429,11 +430,8 @@ object DateUtil {
    * @param date a [[GregorianCalendar]].
    * @return a Julian Day Number.
    */
-  def convertDateToJulianDayNumber(date: GregorianCalendar): Int = {
-    val conv = new JDateTime
-    conv.loadFrom(date)
-    conv.getJulianDate.getJulianDayNumber
-  }
+  def convertDateToJulianDayNumber(date: GregorianCalendar): Int =
+    date.toInstant().atZone(ZoneId.systemDefault()).toLocalDate().getLong(JulianFields.JULIAN_DAY).toInt
```

Comment:
You can omit empty parenthesis and simply call it like that:
`date.toInstant.atZone(ZoneId.systemDefault).toLocalDate.getLong(JulianFields.JULIAN_DAY).toInt`
instead of:
`date.toInstant().atZone(ZoneId.systemDefault()).toLocalDate().getLong(JulianFields.JULIAN_DAY).toInt`